data/data-pipeline/data_pipeline/etl/base.py

```diff
@@ -27,72 +36,68 @@ class ExtractTransformLoad:
     """
 
     APP_ROOT: Path = settings.APP_ROOT
+
+    # Directories
     DATA_PATH: Path = APP_ROOT / "data"
     TMP_PATH: Path = DATA_PATH / "tmp"
-    FILES_PATH: Path = settings.APP_ROOT / "files"
+
+    # Parameters
     GEOID_FIELD_NAME: str = "GEOID10"
     GEOID_TRACT_FIELD_NAME: str = "GEOID10_TRACT"
 
+    # Parameters that will be changed by children of the class
+    # NAME is used to create output path and populate logger info.
+    NAME: str = None
+
+    # LAST_UPDATED_YEAR is used to create output path.
+    LAST_UPDATED_YEAR: int = None
+
+    # SOURCE_URL is used to extract source data in extract().
+    SOURCE_URL: str = None
+
+    # GEO_LEVEL is used to identify whether output data is at the unit of the tract or
+    # census block group.
+    # TODO: add tests that enforce seeing the expected geographic identifier field
+    #  in the output file based on this geography level.
+    GEO_LEVEL: ValidGeoLevel = None
+
+    # COLUMNS_TO_KEEP to used to identify which columns to keep in the output df.
+    COLUMNS_TO_KEEP: typing.List[str] = None
+
+    # Thirteen digits in a census block group ID.
+    EXPECTED_CENSUS_BLOCK_GROUPS_CHARACTER_LENGTH: int = 13
     # TODO: investigate. Census says there are only 217,740 CBGs in the US. This might
     #  be from CBGs at different time periods.
     EXPECTED_MAX_CENSUS_BLOCK_GROUPS: int = 250000
 
+    # Eleven digits in a census tract ID.
+    EXPECTED_CENSUS_TRACTS_CHARACTER_LENGTH: int = 11
     # TODO: investigate. Census says there are only 74,134 tracts in the US,
     #  Puerto Rico, and island areas. This might be from tracts at different time
     #  periods. https://github.com/usds/justice40-tool/issues/964
     EXPECTED_MAX_CENSUS_TRACTS: int = 74160
```

Comment:
very annoying QQ -- wondering if we can set this in a constants file rather than in the class itself, with the idea that if the data changes, it will be less annoying to change? no strong feelings though.

Reply:
Good idea. Created this to track: https://github.com/usds/justice40-tool/issues/1245